frontend/coprs_frontend/coprs/logic/coprs_logic.py

```diff
@@ -292,6 +292,10 @@ def add(cls, user, name, selected_chroots, repos=None, description=None,
         # form validation checks for duplicates
         cls.new(user, name, group, check_for_duplicates=check_for_duplicates)
 
+        delete_after_days = kwargs.get("delete_after_days")
+        if delete_after_days is not None:
+            kwargs["unlisted_on_hp"] = bool(delete_after_days) or kwargs.get("unlisted_on_hp", False)
+
```

Comment:
Interesting approach but I think this won't work when:

1. I create a new project that is going to be deleted in 30 days
2. Update the project and make it not temporary anymore

With this PR, the project will be forever unlisted?

I was thinking more of an approach where we filter temporary projects when querying from the database. Would that work?

Reply:
> With this PR, the project will be forever unlisted?

then you need to uncheck the `unlisted_on_hp` in project settings. But yep, filtering tmp project from DB sounds better. PTAL

Reply:
> then you need to uncheck the unlisted_on_hp in project settings.

Oh, right. I didn't think of that.

>  filtering tmp project from DB sounds better. PTAL

Thank you for the update. LGTM now.